docs/design/README.md

```diff
@@ -443,11 +443,11 @@ programming error:
     to a mathematically incorrect result, such as a two's complement result or
     zero.
 
-The unsigned-integer types are written `uN` or `Carbon.UInt(N)`, with `N` a
-positive multiple of 8. Unsigned integer types wrap around on overflow; we
-strongly advise that they are not used except when those semantics are desired.
-These types are intended for bit manipulation or modular arithmetic as often
-found in [hashing](https://en.wikipedia.org/wiki/Hash_function),
+The unsigned-integer types are written `uN`, with `N` a positive multiple of 8.
```

Comment:
The contrast between "may be written" above (with an example that `i32` is the same as some library type) and "are written" here suggests to me that `iN` is an alias for a library type but `uN` is not. Can we make the phrasing here consistent?

Reply:
I made them both say "may be written"